### PR TITLE
cicd check for npm publish

### PR DESCRIPTION
**Pull request by bhumilsarvaiya**
relates to https://github.com/supertokens/supertokens-website/issues/63

### CODE REVIEW

File: .circleci/publish.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version=$(cat ../package.json | jq .version | tr -d '"')
+isLatest=`curl -s -X GET \
+"https://api.supertokens.io/0/frontend/latest/check?password=$SUPERTOKENS_API_KEY&version=$version&name=website" \
+-H 'api-version: 0'`
+if [[ `echo $isLatest | jq .isLatest` == "true" ]]
+then
+    cd ..
+    npm publish --tag latest
```
Comment (rishabhpoddar) [inline .circleci/publish.sh:8]: Shouldn't you add `latest` and `version-$version` tag?

File: .circleci/publish.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+else
+    cd ..
+    npm publish --tag version-$version
```
Comment (rishabhpoddar) [inline .circleci/publish.sh:11]: Why do we need to prepend with `version-`?

**Comment by rishabhpoddar** [comment]
@bhumilsarvaiya
- I have review this.. added a few comments - they apply to other PRs as well. Please see.
- How will you test this?

File: .circleci/publish.sh (hunk 2 above)
Comment (bhumilsarvaiya) [reply .circleci/publish.sh:11]: a tag name must not be a valid SemVer (Semantic versioning) range (example 2.0.1 or v2.1, etc..)

Comment (bhumilsarvaiya) [reply .circleci/publish.sh:11]: Screenshot 2021-07-22 at 10 41 48 AM

**Comment by rishabhpoddar** [comment]
Okay! Please feel free to merge this and the other branches if you have tested it properly.

**Pull request merged**